recipes/opentelemetry-instrumentation-aiohttp-client/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "opentelemetry-instrumentation-aiohttp-client" %}
+{% set version = "0.17b0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: eca8cdca5b05dc125c68210e813fb5e675b78287fb7645c7ce1dd0b045ff8b3f
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+  run:
+    - aiohttp >=3.0.0,<4.0.0
+    - opentelemetry-api ==0.17b0
```

Comment:
Should we make use of the jinja version bits here?

Reply:
👍